src/websocketClient.ts

```diff
@@ -1,9 +1,11 @@
 import { pipe, prop } from "ramda";
-import * as WebSocket from "ws";
+import * as ReconnectingWS from "reconnecting-websocket";
+import ReconnectingWebsocket from "reconnecting-websocket";
+import * as WS from "ws";
 
 export type Listener = (data: IWebsocketData) => void;
 export type EventListener = (...args: any[]) => void;
-export type MessageListener = (event: IWebsocketMessageEvent) => void;
+export type MessageListener = (event: MessageEvent) => void;
```

Comment:
Should we just kill `IWebsocketMessageEvent` while we're at it? It's not used anywhere else, including FluxCore, FluxWeb, and FluxAPI.

Also this `MessageEvent` being ambiently available rather than imported scares me a little but I guess it's a TypeScript builtin...?

Reply:
haha... we both asked the same question... I'll delete mine.

Reply:
It's basically the type the that the `reconnecting-websocket` is expecting. It doesn't hurt to keep it there *shrug*
And yeah, it's part of the node typescript lib of available.. stuffs

Reply:
Ahaha I noticed that :-P

As far as the library being public -- ours really shouldn't be at this point, honestly. Looks like we'll need to duplicate the repo since this is currently still a fork of my public repo. Not super-duper urgent but should probably still happen soon.